server/src/utils/emailHelper.js

```diff
@@ -99,8 +99,9 @@ const sendEmail = async ({ toAddress, subject, text, html }) => {
       text,
       html,
     };
-
+    console.time();
```

Comment:
What do we need this for?

Reply:
lol we don't.. I'll take it off.
I was testing something yesterday and that just snuck in with my other commits.